model/certificate.go

```diff
@@ -96,7 +94,8 @@ func NewCertificate(domains []string, email string) (*Certificate, error) {
 		Secret:     secret,
 		Domains:    domains,
 		CommonName: common,
-		ACMEEmail:  e.Address,
+		RenewAt:    DefaultRenewAt,
+    ACMEEmail:  e.Address,
```

Comment:
Nitpick but the merge seems to have messed up the tab-scoping

Reply:
the in-browser merge conflict editor isn't great about tabs vs spaces. :P